charts/artifact-hub/templates/scanner_cronjob.yaml

```diff
@@ -32,11 +32,26 @@ spec:
           - name: scanner
             image: {{ .Values.scanner.cronjob.image.repository }}:{{ .Values.imageTag }}
             imagePullPolicy: {{ .Values.pullPolicy }}
+            {{- if .Values.scanner.cacheDir }}
+            env:
+              - name: TRIVY_CACHE_DIR
+                value: {{ .Values.scanner.cacheDir | quote }}
+              - name: XDG_CACHE_HOME
```

Comment:
Have you been able to test that the `scanner` really needs the `XDG_CACHE_HOME`? Both the `hub` and the `tracker` do some operations that rely on Helm libraries, that are probably creating that temporary data. But I can't think of anything in the `scanner` that does it (although I might be overlooking something, hence the question :). I'd just like to make sure we add this where we really need it.

Reply:
You are right, looks like the scanner isn't using helm at all. I removed the XDG_CACHE_HOME setting.